sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -3635,6 +3650,25 @@ class AstBuilder extends DataTypeAstBuilder
     }
   }
 
+  /**
+   * Visits a stringLit context and returns all singleStringLit tokens as an array.
+   *
+   * Returns all tokens without pre-processing. The caller is responsible for applying
+   * config-aware escape processing and concatenation.
+   */
+  override def visitStringLit(ctx: StringLitContext): Array[Token] = {
```

Comment:
This override seems unnecessary, as it changes nothing from the parent implementation.